src/ui/public/visualize/loader/embedded_visualize_handler.js

```diff
@@ -71,6 +72,7 @@ export class EmbeddedVisualizeHandler {
     this._vis.on('update', this._handleVisUpdate);
     this._vis.on('reload', this._reloadVis);
     this._uiState.on('change', this._fetchAndRender);
+    timefilter.on('autoRefreshFetch', this._fetchAndRender);
```

Comment:
I'm not 100% sure what the `forceFetch` parameter does, but I think it means it won't refetch data unless the vis params has changed so we probably want to use `this._reloadVis` as the listener here, just like we do for the `reload` event on `this._vis`.

Reply:
Yeah I would agree with Spencer here. `forceFetch` will actually be passed to the request handler, and it should use it to ignore every possible caching it has. In case of the courier request handler, that means, it won't check if any configuration/timeRange, etc actually changed, and instead always query again. So we will need this for auto refresh, since in case of auto refresh no configuration change, so please replace that by calling `this._reloadVis`.